packages/client/src/addresses/deleteAddress.ts

```diff
@@ -4,17 +4,12 @@ import type { DeleteAddress } from './types';
 /**
  * Responsible for removing the address with the specified 'id'.
  *
- * @function deleteAddress
- * @memberof module:addresses/client
+ * @param props        -
+ * @param props.id     - Identifier of the address.
```

Comment:
Since the feature is being introduced, what about having these errors/warnings solved from the get-go? 😄 

![image](https://user-images.githubusercontent.com/5464401/171593056-be65bffd-bea4-445f-bd91-e0fdf0a1fc8f.png)

Reply:
Fixed